docs/github/v2/06-remote-builder.md

```diff
@@ -0,0 +1,123 @@
+# Remote builder
+
+## What is it?
+
+GitHub Actions run on build machines provided by GitHub. You can see info about the specifications of the build machines [here]
+(https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners).
+
+For Unity projects the disc size is quite small. You may experience an error related to running out of disc space. You may also want to run the build on a server with more memory or CPU resources.
```

Comment:
```suggestion
For Unity projects the available disk size is quite small. You may experience an error related to running out of disk space. You may also want to run the build on a server with more memory or processing resources.
```

Source for [disk](https://www.oxfordlearnersdictionaries.com/definition/english/disc#:~:text=The%20usual%20spelling%20is%20disc,even%20in%20(British%20English).)